credoai/modules/dataset_modules/dataset_base.py

```diff
@@ -24,8 +24,10 @@ def run(self):
         pipe = self._make_pipe()
         cv_results = self._run_cv(pipe)
         group_differences = self._group_differences()
+        feature_space_proxy_scores = self._assess_proxy_in_feature_space()
         self.results = {'sensitive_feature_roc_auc': cv_results.mean(),
-                        'group_diffs': group_differences}  
+                        'group_diffs': group_differences,
```

Comment:
`group_differences` fails to be json serializable I believe. I'd check that.

Reply:
This still needs to be addressed for the dataset assessment to work I think

Reply:
Is there anything I should do for this here?

Reply:
You just need to make sure the output is serializable. I don't believe t currently is - you'll have to rewrite group differences. I think you may need to convert the series it outputs into a dict? Not sure.

Reply:
done